governor/src/gcra.rs

```diff
@@ -79,22 +79,21 @@ pub(crate) struct Gcra {
     /// The "weight" of a single packet in units of time.
     t: Nanos,
 
-    /// The "burst capacity" of the bucket.
+    /// The "tolerance" of the bucket in units of time.
+    ///
+    /// The total "burst capacity" of the bucket is `t + tau`.
     tau: Nanos,
 }
 
 impl Gcra {
     pub(crate) fn new(quota: Quota) -> Self {
-        let tau: Nanos = (cmp::max(quota.replenish_1_per, Duration::from_nanos(1))
-            * quota.max_burst.get())
-        .into();
-        let t: Nanos = quota.replenish_1_per.into();
+        let t: Nanos = cmp::max(quota.replenish_1_per, Duration::from_nanos(1)).into();
+        let tau: Nanos = t * (quota.max_burst.get() - 1).into();
```

Comment:
Hm, hm, hm. I think this might cause `tau` to become 0 if you pick a burst capacity of 1... I'm slightly convinced that won't cause problems in the `tat` calculation below, but not fully certain. a `tau=0` case is fine, right?

Reply:
Oh, I forgot to answer your actual question: yes, `tau=0`  should be fine in the decision calculations.

What I'm less sure about is if I adjusted all places where `tau` is used outside of the `gcra` module... in fact, let me take another look today and see if I didn't missed anything the tests also didn't catch.